### PR TITLE
Fixes a few grammar mistakes

### PR DESCRIPTION
**Pull request by legoboy0215**
IDK. English is not my native tongue.

### CODE REVIEW

**Comment by Muqsit** [comment]
+1

**Comment by Muqsit** [comment]
Approved!

**Comment by ishitatsuyuki** [comment]
Rejected: some are correct, some are wrong. We should keep it for now.

File: README.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ See [wiki](https://github.com/iTXTech/Genisys/wiki/Use-Docker-to-run-Genisys) fo
 * One Core to rule anything
 * This core is an unofficial version of PocketMine-MP modified by DREAM STUDIO and iTX Technologies LLC.
 * Genisys is only a fork of PocketMine-MP and all original codes are written by PocketMine Team.
-* [Download php7 for Genisys](https://github.com/iTXTech/PHP-Genisys/)
+* [Download PHP7 for Genisys](https://github.com/iTXTech/PHP-Genisys/)
 * [Download Genisys Installer for Windows](https://raw.githubusercontent.com/iTXTech/Genisys-Installer/master/setup.exe)
-* Feel free to create a Pull Request or open an Issue. English and Chinese are both welcome. Use English to communicate with more people.
+* Feel free to create a Pull Request or open an Issue. English or Chinese are both accepted. Use English to communicate with more people.
```
Comment (ishitatsuyuki) [inline README.md]: I don't think it's "or".

File: README.md [hunk 2]
```diff
@@ -41,20 +41,20 @@ See [wiki](https://github.com/iTXTech/Genisys/wiki/Use-Docker-to-run-Genisys) fo
...
+Most codes are written by PocketMine team and licensed under GPLv3. Some of the AI is proprietary, copying is prohibited.
 
 ### Official Development Documentation
 [Genisys Official Development Documentation Page](http://docs.mcper.cn/en-US/)
 
 ### Features
-* Performance optimization (Let 100+ players join in a server)
-* Bug fix in PocketMine-MP
+* Performance optimization (Lets 100+ players join in a server)
+* Lots of bug fixes in PocketMine-MP
```
Comment (ishitatsuyuki) [inline README.md]: This is unneeded.

File: README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ Most codes are made by PocketMine team and licensed under GPLv3. Some AI is prop
 * More Items
 * Hunger (Based on Katana)
 * AI (Based on MOW)
-* More commands
+* More commands than the original PocketMine-MP
```
Comment (ishitatsuyuki) [inline README.md]: Unneeded.

File: README.md [hunk 4]
```diff
@@ -41,20 +41,20 @@ See [wiki](https://github.com/iTXTech/Genisys/wiki/Use-Docker-to-run-Genisys) fo
...
-* Base food system is based on **Katana**
+* Base hunger/food system is based on **Katana**
 * Base weather system was written by **@Zzm317** and rewritten by **@PeratX**
 * **@FENGberd**'s encouragement
-* Our leaders are **@ishitatsuyuki** and **@jasonczc**
+* Our project leaders are **@ishitatsuyuki** and **@jasonczc**
 
 ### License
-Most codes are made by PocketMine team and licensed under GPLv3. Some AI is proprietary, copy is prohibited.
+Most codes are written by PocketMine team and licensed under GPLv3. Some of the AI is proprietary, copying is prohibited.
```
Comment (ishitatsuyuki) [inline README.md]: AI is countable. I think it should be Some AIs.

**Comment by gaoyichuan** [comment]
No need. Rejected

**Comment by legoboy0215** [comment]
Well, then close it XD

File: README.md (hunk 1 above)
Comment (CLiu13) [inline README.md]: It is "and".

**Pull request closed without merging**